.github/workflows/R-CMD-check.yaml

```diff
@@ -51,6 +40,24 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@master
 
+      - name: Brew and macOS config
+        if: runner.os == 'macOS'
+        run: |
+          brew install pkg-config
+          brew install udunits
+          brew install gdal
+          echo "PKG_CONFIG_PATH=/usr/local/lib/pkgconfig/" >> .Renviron
+          echo "PROJ_LIB=/usr/local/opt/proj/share/proj/" >> .Renviron
+          cat <<EOT >> .Rprofile
+          r <- getOption("repos")
+          r["CRAN"] <- "https://cran.rstudio.com"
+          r["rccp_drat"] <- "https://RcppCore.github.io/drat"
+          config_args <- c("sf" = "--with-proj-lib=/usr/local/lib/", "rgdal" = "--with-proj-lib=/usr/local/lib/ --with-proj-include=/usr/local/include/")
+          options(configure.args = config_args,
+                  install.packages.compile.from.source = "yes",
+                  repos = r)
+          EOT
```

Comment:
For my own education, what is `EOT` and `cat <<EOT >> .Rprofile` (LN51) for?

Reply:
It's called a "[heredoc](https://linuxhint.com/bash-heredoc-tutorial/)" - basically says to send all lines between `<<EOT` (L51) and `EOT` (L59) into the file `.Rprofile`.